Sources/LambdaConverters.Wpf/RuleErrorStrategy.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Windows.Controls;
+
+namespace LambdaConverters
+{
+    /// <summary>
+    /// Defines the validation rule error strategy.
+    /// </summary>
+    public enum RuleErrorStrategy
+    {
+        /// <summary>
+        /// The default of <see cref="ValidationRule" /> is returned.
+        /// </summary>
+        ReturnDefaultValue,
```

Comment:
_Default value_ is somewhat misleading. Default value of a reference type is always null. Please rename the enum field to `ReturnNull`.